resolver/resolver.cc

```diff
@@ -385,8 +407,20 @@ class ResolveConstantsWalk {
 
         // Determine if a package with the same name as `scope` is known to export the name `cnst`.
         bool packageExportsConstant(const std::vector<core::NameRef> &scope, core::NameRef cnst) const {
-            const auto parent = absl::c_find_if(this->parents, [&scope](auto &p) { return p.stub.fullName == scope; });
-            return parent != this->parents.end() && parent->exportsSymbol(cnst);
+            ENFORCE(!scope.empty());
+
+            auto start = scope.begin();
+            if (*start == core::Names::Constants::Test()) {
+                ++start;
+            }
+
+            auto scopeSize = std::distance(start, scope.end());
```

Comment:
Should this also be `ENFORCE`'d to be non-zero?  (That is, we should always see non-empty scopes even after stripping a top-level `Test` namespace.)

Reply:
Happy to add that. I think that it's sort of hand-wavingly implied right now by errors being present when trying to define a package named `Test`, but we might as well be explicit here.